modules/importing-manifest-list-through-imagestreamimport.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+// * assembly/openshift_images/managing-image-streams.adoc
+
+:_content-type: PROCEDURE
+[id="importing-manifest-list-through-imagestreamimport_{context}"]
+= Importing a manifest list through ImageStreamImport
+
+Use the following procedure to import a manifest list through the `ImageStreamImport` object with the `importMode` value. 
+
+.Procedure
+
+. Create an `ImageStreamImport` object that sets the `importMode` to `Manifestlist` by running the following command:
++
+[source,terminal]
+----
+$ cat <<EOF | oc create -f -
+apiVersion: image.openshift.io/v1
+kind: ImageStreamImport
+metadata:
+  name: app
+  namespace: <namespace>
+spec:
+  import: true
+  images:
+  - from:
+      kind: DockerImage
+      name: quay.io/<user_name>/<image_name>
+    to:
+      name: latest
+    referencePolicy:
+      type: Source
+    importPolicy:
+      importMode: "ManifestList"
+EOF
+----
+
+. Enter the following command to ensure that Image objects were created for the manifest list: 
++
+[source,terminal]
+----
+$ oc get image | grep quay.io/<user_name>/<image_name>
+----
+
+. Enter the following command to ensure that all sub-manifests were created:
++
+[source,terminal]
+----
+$ oc get is app
+----
++
+[NOTE]
+====
+You can compare the output from the aforementioned commands against the content of `quay.io/<user_name>/<image_name>`. If correct, there should be one image for the manifest list, and one image for each sub-manifest. 
+====
+
+. (Optiona) Delete the images previously created by entering the following command:
++
+[source,terminal]
+----
+$ oc delete images $(oc get images | grep quay.io/<user_name>/<image_name> | awk '{ print $1 }')
```

Comment:
Please don't mention `oc delete images` in docs. It creates an illusion that the image is deleted, but only `oc adm prune images` can correctly delete image objects

* without breaking existing image streams
* with the removal of unnecessary data from the image registry storage

If you delete an image object via `oc delete` you can have leftovers on the storage that will never be deleted.

Reply:
@dmage should I replace `oc delete images` with `oc adm prune images $(oc adm prune images | grep quay.io/<user_name>/<image_name> | awk '{ print $1 }')` ?

Reply:
The pruner doesn't have any parameters, it works with all objects. And the pruner can be used only by admins, while everybody can import an image. So

* delete image stream tags/image streams that you don't need
* wait for some time (by default - 1 hour)
* run the pruner `oc adm prune images`

The wait time is also tricky, it can be changed via `--keep-younger-than`, but if you set it to a low value (like 5 seconds), this may corrupt ongoing image pushes.

So overall - normal users cannot interact with `image` objects, and admins shouldn't interact with them directly unless they really know what they are doing.